sam-hiberry-app/create_order/app.py

```diff
@@ -0,0 +1,50 @@
+import json
+from datetime import datetime
+import uuid
+import logging
+
+# Initialize logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+    """
+    """
+
+    try:
+        data = json.loads(event['body'])
+
+        order = {
+            "id": str(uuid.uuid4()),  
+            "client_name": data.get("client_name"),
+            "delivery_date": data.get("delivery_date"),  
+            "delivery_time": data.get("delivery_time"),
+            "address": data.get("address"),
+            "latitude": float(data.get("latitude", 0)),
+            "longitude": float(data.get("longitude", 0)),
+            "phone_number": data.get("phone_number"),
+            "cart_items": data.get("cart_items", []),  
+            "total_amount": float(data.get("total_amount", 0)),
+            "payment_method": data.get("payment_method"),
+            "created_by": int(data.get("created_by", 0)),
+            "created_at": data.get("created_at", datetime.now().isoformat()),
+            "updated_by": int(data.get("updated_by", 0)),
+            "updated_at": data.get("updated_at", datetime.now().isoformat()),
+            #"errors": data.get("errors", []), 
+            #"notes": data.get("notes"),
+            #"status": data.get("status"),
+            #"delivery_sequence": int(data.get("delivery_sequence", 0))
+        }
+        
+        print(f'Order received and created: {order}')
+        return {
+            'statusCode': 200,
+            'body': json.dumps('Order received and created successfully.')
+        }
+
+    except Exception as e:
```

Comment:
i would suggest adding more exceptions, for validations or for unauthorized access, so we can return a 404, 400 or 403